Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Internal/CustomerCancellationDues.hs

```diff
@@ -88,7 +89,9 @@ disputeCancellationDues merchantId merchantCity apiKey CancellationDuesReq {..}
         logWarning "Unable to calculate dispute chances used"
         return 0
       else return $ round $ disputeAmount / transporterConfig.cancellationFee
+  let numOfCancellations = disputeChances
```

Comment:
I think you can just use `disputeChances` instead of a new variable

Reply:
won't it create confusion while reading the code like we can do this